bigflow/workflow.py

```diff
@@ -29,8 +30,34 @@ def daily_start_time(start_time: dt.datetime) -> dt.datetime:
     return start_time.replace(hour=0, minute=0, second=0, microsecond=0) - td
 
 
+class JobContext(typing.NamedTuple):
+    runtime: typing.Optional[dt.datetime]
+    runtime_raw: typing.Optional[str]
+    workflow: typing.Optional['Workflow']
+    workflow_id: typing.Optional[str]
```

Comment:
why not workflow.workflow_id ?

Reply:
I was assuming (incorrectly) hat it might be a situation when we don't have a reference to the Workflow, but know it's id.
I've removed 'workflow_id' from JobContext